docs/userguide.rst

```diff
@@ -0,0 +1,217 @@
+.. _userguide:
+
+User guide
+==========
+
+.. admonition:: Work-In-Progress
+
+   **FIXME** The ``reana-client`` package is a not-yet-released work in
+   progress. Moreover, the ``reana-server`` instance at reana.cern.ch is
+   still not available.
+
+Verify REANA client connection
+------------------------------
+
+In order to check if the ``reana-client`` can connect with REANA cloud,
+we can run the following command:
+
+.. code-block:: console
+
+   $ reana-client ping
+   [INFO] REANA Server URL set to: http://reana.cern.ch
+   [INFO] Connecting to http://reana.cern.ch
+   [INFO] Server is running.
+
+
+Validating analyses specifications
+----------------------------------
+
+.. admonition:: Work-In-Progress
+
+   **FIXME** Not implemented yet.
+
+You can validate your current analysis specification (``./.reana.yaml``), so it
+can be run in REANA, as follows:
+
+.. code-block:: console
+
+   $ reana-client analysis validate
+   File .reana.yaml is a valid REANA specification file.
```

Comment:
`./reana.yaml`

Reply:
Both fixed, although this PR is again primarily about restructuring and features, we need to update content heavily, e.g. the current implementation is:

```console
$ reana-client analyses analysis_validate
``` 

(which BTW reads a bit weird; to be updated)